src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,36 @@
+@import '/src/styles/blocks/variables';
+
+.stars {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+  margin: 0 0 24px;
+
+  &__total {
+    display: flex;
+  }
+
+  &__number-Of-Reviews {
+    color: rgba(6, 11, 53, 1);
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    margin: 0;
+  }
+
+  &__total--4 div:nth-child(-n + 4) {
+    background-image: url(/src/images/star-active.svg);
+    background-position: center;
+    background-repeat: no-repeat;
+  }
```

Comment:
The class name `&__total--4` does not follow BEM naming convention. The modifier value should not be in the class name. It would be better to use a class name like `stars--4` and apply it to the parent block. Then, you can style the child elements based on this class.